Uno/NugetPackageExplorer.Legacy/Presentation/Content/InspectPackageViewModel.cs

```diff
@@ -268,6 +271,32 @@ public static async Task<InspectPackageViewModel> CreateFromRemotePackageWithFal
             }
         }
 
+        public async Task DoubleClick()
+        {
+            if (SelectedContent is IFile file)
```

Comment:
I would wrap this entire method in `#if __WASM__ || __MACOS__` or `#if !HAS_UNO_SKIA_GTK`, [because FileSavePicker arnt supported yet for GTK](https://github.com/unoplatform/uno/blob/master/doc/articles/features/windows-storage-pickers.md). While no one uses NPE on that platform atm, but it is better to be safe.

Reply:
I guess it will be supported soon: https://github.com/unoplatform/uno/pull/6544
But yes, I could #if it for now.